doc/source/rest.j2

```diff
@@ -551,15 +551,85 @@ argument, and in this case the second argument passed is the value, or it.
 The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
 supported, and take a list of arguments as parameters.
 
+.. _aggregates:
+
+Aggregates: on the fly, measurements modification and aggregatation
+===================================================================
+
+Gnocchi allows to do on-the-fly aggregation and modification of already
+aggregated data of |metrics|.
+
+It can be done by providing the list of |metrics| to aggregate:
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiples it by `4`.
+
+.. Note:
+
+   `operations` can also be passed as a lisp string, for this example:
+   `"operations": "(aggregate mean (metric all))"`
+
+Operations between metrics can also be done, such as:
+
+{{ scenarios['get-aggregates-between-metrics']['doc'] }}
+
+Partial list of supported maths/aggregation::
+
+    ```
+    (rolling mean 2 (...))
+    (aggregate min (...))
+    (resample sum 60 (...))
+    (absolute (...))
+    (negative (...))
+    (>= (...) 2)
+    (* (...) 5)
+    (mod (...) 2)
+    ```
+
+Also, aggregation across |metrics| have different behavior depending
+on whether boundary values are set ('start' and 'stop') and if 'needed_overlap'
+is set.
+
+Gnocchi expects that we have a certain percent of timestamps common between
```

Comment:
Gnocchi expects that time series have a certain percentage of timestamps in common.